lib/msf/ui/console/command_dispatcher/auxiliary.rb

```diff
@@ -48,9 +55,25 @@ def method_missing(meth, *args)
 
       return mod.send(meth.to_s, *args)
     end
+
+    method = meth.to_s.split(/_/)
+    mod_actions = Serializer::ReadableText.dump_module_actions(mod, '   ')
+    if mod && mod.kind_of?(Msf::Module::HasActions) && mod_actions.include?(method[1])
+       do_action(method[1], *args)
+    end
```

Comment:
A couple things here. Since the action name may have an `_` in it, we should use the `String#delete_prefix` method to make sure we get the whole action name and that only a prefix of `cmd_` prefix is deleted. Also, instead of generating a whole table each time, we can check the action name against the list straight from the module.
```suggestion
    action = meth.to_s.delete_prefix('cmd_')
    mod_actions = mod.actions.map(&:name)
    if mod && mod.kind_of?(Msf::Module::HasActions) && mod_actions.include?(action)
       do_action(action, *args)
    end
```

Reply:
Thank you very much! I'll made the improvement